<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Giri≈ü</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #2C3E50,
        stop: 1 #34495E
    );
    color: white;
    font-family: 'Segoe UI', sans-serif;
}

QLabel {
    color: white;
    font-size: 16px;
    background: transparent;
}

QLineEdit {
    background: rgba(44, 62, 80, 0.7);
    border: 2px solid transparent;
    border-radius: 10px;
    padding: 10px;
    color: white;
    font-size: 14px;
    selection-background-color: #3498DB;
}

QLineEdit:hover {
    border-color: #3498DB;
}

QLineEdit:focus {
    border-color: #2980B9;
    background: rgba(44, 62, 80, 0.9);
}

QPushButton {
    background: #3498DB;
    border: none;
    border-radius: 10px;
    padding: 12px;
    color: white;
    font-weight: bold;
    font-size: 14px;
}

QPushButton:hover {
    background: #2980B9;
    margin-top: -2px;
    margin-bottom: 2px;
}

QPushButton:pressed {
    background: #2574A9;
    margin-top: 0px;
    margin-bottom: 0px;
}

QCheckBox {
    color: white;
    font-size: 14px;
    spacing: 8px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border-radius: 5px;
    border: 2px solid #3498DB;
    background: rgba(44, 62, 80, 0.7);
}

QCheckBox::indicator:hover {
    border-color: #2980B9;
}

QCheckBox::indicator:checked {
    background: #3498DB;
    color: white;
    font-family: 'Segoe UI', sans-serif;
    font-size: 14px;
    content: "‚úì";
}

QCheckBox::indicator:checked:hover {
    background: #2980B9;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <widget class="QLabel" name="logo">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 48px;
    font-weight: bold;
    color: white;
    background: rgba(44, 62, 80, 0.7);
    border-radius: 15px;
    padding: 20px;
}</string>
     </property>
     <property name="text">
      <string>üõ∞Ô∏è</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="title">
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 24px;
    font-weight: bold;
    color: white;
    background: transparent;
}</string>
     </property>
     <property name="text">
      <string>Yer ƒ∞stasyonu</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="username">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Kullanƒ±cƒ± Adƒ±</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>≈ûifre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="remember_me">
     <property name="text">
      <string>Beni Hatƒ±rla</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="login_btn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="text">
      <string>Giri≈ü Yap</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 